exporter/datadogexporter/internal/translator/metrics_translator.go

```diff
@@ -210,7 +212,7 @@ func (t *Translator) getSketchBuckets(
 		count := p.BucketCounts()[j]
 		if delta {
 			as.InsertInterpolate(lowerBound, upperBound, uint(count))
-		} else if dx, ok := t.prevPts.putAndGetDiff(name, bucketTags, ts, float64(count)); ok {
+		} else if dx, ok := t.prevPts.Diff(name, bucketTags, startTs, ts, float64(count)); ok {
```

Comment:
This could be `MonotonicDiff`, since the bucket count values should be increasing. However, in the case of a bug downstream, it could lead to reseting some bucket count values but not others, making the error harder to debug